website/src/utils.js

```diff
@@ -1,5 +1,6 @@
 var nodemailer = require('nodemailer');
 var nconf = require('nconf');
+nconf.file({ file: './config.json' });
```

Comment:
why this? we already load the file here https://github.com/crookedneighbor/habitrpg/blob/config-grunt/website/src/utils.js#L120